contracts/Tokens/VTokens/VTokenInterfaces.sol

```diff
@@ -254,7 +254,7 @@ contract VTokenInterface is VTokenStorage {
     /**
      * @notice Event emitted when the reserves are reduced
      */
-    event ReservesReduced(address admin, uint reduceAmount, uint newTotalReserves);
+    event ReservesReduced(address protocolShareReserve, uint reduceAmount, uint newTotalReserves);
```

Comment:
I would use something like `receiver` instead of `protocolShareReserve` in the event defintion